api/src/main/java/com/velocitypowered/api/event/EventManager.java

```diff
@@ -19,10 +19,11 @@
      * Requests that the specified {@code handler} listen for events and associate it with the {@code plugin}.
      * @param plugin the plugin to associate with the handler
      * @param eventClass the class for the event handler to register
+     * @param postOrder the order in which events should be posted to the handler
      * @param handler the handler to register
      * @param <E> the event type to handle
      */
-    <E> void register(@NonNull Object plugin, @NonNull Class<E> eventClass, @NonNull EventHandler<E> handler);
+    <E> void register(@NonNull Object plugin, @NonNull Class<E> eventClass, @NonNull PostOrder postOrder, @NonNull EventHandler<E> handler);
```

Comment:
Would like an overload without the `PostOrder` that simply specifies a normal post order.